src/pages/a-complete-guide-to-useeffect/index.es.md

```diff
@@ -0,0 +1,1673 @@
+---
+title: Guía completa de useEffect
+date: '2019-03-09'
+spoiler: Los Effects son parte de tu flujo de datos
+cta: 'react'
+---
+
+Escribiste algunos componentes con [Hooks](https://reactjs.org/docs/hooks-intro.html). Talvez incluso una pequeña aplicación. Te sientes satisfecho. Te sientes cómodo con el API e incluso pudiste aplicar algunos trucos mientras la desarrollabas. Incluso creaste algunos [Hooks personalizados](https://reactjs.org/docs/hooks-custom.html) para extraer lógica repetitiva (adiós a 300 líneas) y se lo mostraste a tus colegas. "Buen trabajo", dijeron.
```

Comment:
```suggestion
Escribiste algunos componentes con [Hooks](https://reactjs.org/docs/hooks-intro.html). Tal vez incluso una pequeña aplicación. Te sientes satisfecho. Te sientes cómodo con el API e incluso pudiste aplicar algunos trucos mientras la desarrollabas. Incluso creaste algunos [Hooks personalizados](https://reactjs.org/docs/hooks-custom.html) para extraer lógica repetitiva (adiós a 300 líneas) y se lo mostraste a tus colegas. "Buen trabajo", dijeron.
```

Reply:
While _talvez_ is an accepted form and even more extended in South America than the spaced _tal vez_, the latter is more correct.

Reply:
[Relevant article](https://www.fundeu.es/recomendacion/tal-vez-talvez/) explaining why _tal vez_ is the preferred form. Thank you for the Spanish translation 😊

Reply:
Thank you, @d0vi, you are right. I just fixed 7 instances of this word :smile: